src/yoroi-wallets/types/types.ts

```diff
@@ -57,3 +57,24 @@ export type YoroiAmount = {
 export type YoroiMetadata = {
   [label: string]: string
 }
+
+export type YoroiNFT = {
+  id: string
+  name: string
+  description: string
+  image: string
+  thumbnail: string
+  metadata: {
+    policyId: string
+    assetNameHex: string
+    originalMetadata: object
+  }
+}
+
+export enum YoroiNFTModerationStatus {
```

Comment:
avoid enums
prefer string unions
`type Status = "red" | "yellow"`
or
```
const Status = {
  RED: "RED",
  YELLOW: "YELLOW"
} as const
type Status = keyof typeof Status

const red: Status = Status.RED```

Reply:
avoid all caps

Reply:
@wolverineks this is taken from https://emurgo.atlassian.net/wiki/spaces/CARDANO/pages/734560416/Moderating+NFT+images and will require us to use all caps at least when parsing the response from rest API. If we want to use `type Status = "red" | "yellow"` we would have to maintain 2 different types and a map from one to the 2nd one. What are your thoughts?

Reply:
as soon as data is received from the server, it should be converted to adhere to the conventions of this repo. the server types should be maintained by the server repo. @stackchain what's the status on server types?

Reply:
https://emurgo.atlassian.net/wiki/spaces/CARDANO/pages/734560416/Moderating+NFT+images